File: exs/mundo_2/python/054.py
"""
Desafio 054

Problema: Crie um programa que leia o ano de nascimento
          de sete pessoas. No final, mostre quantas pessoas
          ainda não atingiram a maioridade e quantas já são maiores.

Resolução do problemas:
"""
from datetime import date

maioresDeIdade = 0
menoresDeIdade = 0
for c in range(7):
    ano_nascimento = int(input('Informe o ano de nascimento da {}º pessoa: '.format(c + 1)))

    if (date.today().year - ano_nascimento) < 21:
        menoresDeIdade += 1
    else:
        maioresDeIdade += 1
print()
print('Tem {} pessoas menores de idade\nE {} pessoas maiores de idade.'.format(menoresDeIdade, maioresDeIdade))

File: exs/mundo_2/ruby/054.rb
=begin
Desafio 054

Problema: Crie um programa que leia o ano de nascimento
          de sete pessoas. No final, mostre quantas pessoas
          ainda não atingiram a maioridade e quantas já são maiores.

Resolução do problemas:
=end

maiores = 0
menores = 0
7.times do |i|
	print"Digite o ano de nascimento da #{i+1} pessoa: "
	anoNasc = gets.chomp.to_i
	idade = Time.now.year - anoNasc
	if idade >= 21
		maiores += 1
	else
		menores += 1
	end
end
puts "Tem #{maiores} pessoas maiores de idade."
puts "Tem #{menores} pessoas menores de idade."
